examples/openvino/resnet50/requirements.txt

```diff
@@ -0,0 +1,3 @@
+datasets
+openvino
```

Comment:
This is not required here. `openvino` package will be installed by `pip install olive-ai[openvino]` which is recommended to install for all openvino features.